src/app/widgets/color_label.cpp

```diff
@@ -46,4 +46,4 @@ void ColorLabel::update(QString color)
     QPixmap block(this->icon->pixmap()->size());
     block.fill(this->theme->get_color(color));
     this->icon->setPixmap(block);
-}
\ No newline at end of file
+}
```

Comment:
add the newline back

Reply:
Not removed in my commit, but newline restored.